backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -42,6 +42,31 @@
 TRACIM_DEFAULT_PERM = "tracim"
 
 
+class TracimSecurityPolicy:
+    """
+    Adapter to keep existing authentification method compatible with pyramid 2.0
+    source:
```

Comment:
```suggestion
    Adapter to keep existing authentification method compatible with pyramid 2.0
    source:
```
Note: we may want to support the new security policy mecanism as we still use deprecated mecanism this way, but i decided for now to just do the necessary to make things work.

Reply:
How long do you think it would take to support it? I think it's a good thing to not keep deprecated mecanisms.

Reply:
It really depend how far we go. We do rely on a package that is not compatible with this for now, so if we do want to support this properly we add to replace it/update it for pyramid 2.0 which may take time.

Also, as we completely skip the authorization mecanism of pyramid, i'm not sure we should still use the authentification mecanism, but this may take some time also.

Reply:
ticket created about this https://github.com/tracim/tracim/issues/5488